### ISSUE TITLE
Beaglebone

### ISSUE THREAD

**Issue by chelliwell** [RobertCNelson/meta-beagleboard-kernel#1]
Which would be the machine for BeagleBone (Black) - I notice 'beaglebone' isn't in the COMPATIBLE_MACHINE?

**Comment by RobertCNelson** [comment]
Yeah, use: https://github.com/resin-io/meta-resin/tree/master/meta-resin-beaglebone/recipes-kernel/linux

This repo is more of manual overlay on top of meta-ti (manually doing meld).

I'll probally be deleting it form the public, just use meta-resin..

Regards,
